### PR TITLE
Merged docker build sections

### PR DESCRIPTION
**Pull request by adarsh-ideal**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message. Fixes https://github.com/loklak/loklak_server/issues/968
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: docs/installation_docker.md [hunk 1]
```diff
@@ -2,21 +2,45 @@
...
 
-- clone the repository and build the image from source
+#### Build a docker image on your local machine
 
-        git clone https://github.com/loklak/loklak_server.git
-        cd loklak_server
-        docker build -t mariobehling/loklak -f docker/Dockerfile .
+1. Type the following commands in terminal
+   ```
+   sudo apt-get update && sudo apt-get upgrade -y
```
Comment (yukiisbored) [inline docs/installation_docker.md]: Not everyone uses Ubuntu or Debian, some uses Fedora, Gentoo, Arch, SUSE, etc.

File: docs/installation_docker.md [hunk 2]
```diff
@@ -2,21 +2,45 @@
...
+   sudo apt-get install docker.io
+   sudo docker build https://github.com/loklak/loklak_server.git
```
Comment (yukiisbored) [inline docs/installation_docker.md]: I don't think this will work because the Dockerfile is not on the root of this repo, it's in the docker folder

File: docs/installation_docker.md (hunk 1 above)
Comment (yukiisbored) [inline docs/installation_docker.md]: What I'm saying is it's better to tell them "install docker" and give them a couple of example commands in how to achieve that on popular distros (primarily Debian/Ubuntu, Fedora, SUSE) rather than saying "type these following commands" because not everyone is running a debian based system and it can lead to confusion for new Linux users.

File: docs/installation_docker.md [hunk 3]
```diff
@@ -238,31 +262,3 @@ Enter your assigned HTTP IP address into your browser to check if it is working.
...
-   sudo apt-get install docker.io
-   sudo docker build https://github.com/loklak/loklak_server.git
-   ```
-
-2. You will get a hash from last command. Copy that hash and do...
-   ```
-   sudo docker tag <IMAGE_HASH> <NAME_FOR_IMAGE>
-   ```
-
-### Automated build on Cloud using GitHub repository
```
Comment (yukiisbored) [inline docs/installation_docker.md]: Why is this removed, docker cloud is still a thing.

**Comment by yukiisbored** [comment]
Your commit message doesn't contain "Fixes/Closes <link to issue or issue num>", please amend and add that.

**Comment by yukiisbored** [comment]
It's also a good idea to use imperative present tense rather than past tense for your commit message.

File: docs/installation_docker.md (hunk 2 above)
Comment (yukiisbored) [inline docs/installation_docker.md]: And also you don't need `sudo` to use docker, just add yourself to the `docker` group (`usermod -aG user docker`)

**Comment by yukiisbored** [comment]
Also, a good practice in using git is to **always** branch out for your changes especially for PRs. So you can fix problems like merge conflicts, bad commits, etc easily.

File: docs/installation_docker.md (hunk 1 above)
Comment (yukiisbored) [inline docs/installation_docker.md]: Ah, sorry. I didn't see this was actually part of actual file. Gonna create a separate issue for this.

File: docs/installation_docker.md (hunk 2 above)
Comment (yukiisbored) [inline docs/installation_docker.md]: Ah, sorry. I didn't see this was actually part of actual file. Gonna create a separate issue for this.

**Comment by jig08** [comment]
Thanks @yukiisbored for your inputs :) 
@adarsh-ideal +1
